flang/test/Driver/cuda-option.f90

```diff
@@ -0,0 +1,13 @@
+! Test -fcuda option
+! RUN: %flang -fc1 -cpp -fcuda -fdebug-unparse %s -o - | FileCheck %s
```

Comment:
Could you add a RUN line without enabling CUDA? Otherwise it's hard to see what's being tested and what the impact of enabling CUDA is.

Reply:
Without it, it would just fail during parsing. Do you want a test that check the failure?

Reply:
Added